internal/contracts/contracts.go

```diff
@@ -274,4 +276,34 @@ func (mc *JSONContract) Unmarshal() (Contract, error) {
 		mc.StateNonce,
 	}
 	return c, nil
+	
+}
+
+
+
+func GenerateRandomContract() (*Contract, error){
+	b := make([]byte, 32)
+	_, err :=rand.Read(b)
```

Comment:
When will this return a non-nil error

Reply:
The read function from rand is a helper function that uses io.readfull function from. The readFull function reads the length of the bytes and returns an error if fewer bytes were read. So, if for some reason not all the bytes were read, it would return an error

Reply:
If there's no identifiable reason that all the bytes will be read, then ignore the error. I don't want to approve a generate random contract function that returns any errors because that is impractical.